src/parsers/psmt2_to_alt_ergo.ml

```diff
@@ -115,8 +115,8 @@ module Translate = struct
     in
     aux sort.ty
 
-  let better_num_of_string s =
-    begin match String.split_on_char '.' s with
+  (* -- unused -- let better_num_of_string s =
```

Comment:
Let's just remove this. It is in the git history if we really need it.

Reply:
Sure but we will remove the legacy parser so I did the minimal effort.